packages/malloy/src/lang/parse-tree-walkers/document-highlight-walker.ts

```diff
@@ -194,6 +194,9 @@ export function passForHighlights(
       case MalloyParser.RENAME:
         register(token, HighlightType.Property.Rename, true);
         break;
+      case MalloyParser.SOURCE:
```

Comment:
Might be worth getting rid of the highlight for `explore` as I don't believe that's a valid keyword anymore.